jenkins_pipelines/scripts/SUSEManager_cleaner/README.md

```diff
@@ -0,0 +1,37 @@
+# SUSE Manager API and SSH Management Script
+
+This Python script provides a command-line interface to manage various resources and configurations on a SUSE Manager server through the XML-RPC API and SSH commands. 
+The script enables testers to perform resource cleanup, SSH-based file management, and update custom repositories remotely.
+
+## Prerequisites
+
+- **Python 3**: Ensure Python 3 is installed.
```

Comment:
I see we use Python 3.11 in CI.
Do we specifically need at least that version or will any (reasonably recent) Python 3.X be fine ?